### PR TITLE
[type:refactor #3088]format shenyu-dict-sqlmap.xml

### PR DESCRIPTION
**Pull request by shallwetalkge**
format shenyu-dict-sqlmap.xml https://github.com/apache/incubator-shenyu/issues/3088

### CODE REVIEW

File: shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml [hunk 1]
```diff
@@ -32,65 +32,89 @@
...
+        date_created,
+        date_updated
     </sql>
 
     <select id="selectById" parameterType="java.lang.String" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from shenyu_dict
-        where id = #{id,jdbcType=VARCHAR}
+        SELECT
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml:48]: The keywords `SELECT` are not right-aligned, it should be set back one space.

File: shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml [hunk 2]
```diff
@@ -32,65 +32,89 @@
...
+         FROM shenyu_dict
+        WHERE dict_code = #{dictCode, jdbcType=VARCHAR}
+          AND dict_name = #{dictName, jdbcType=VARCHAR}
     </select>
 
     <select id="findByType" parameterType="java.lang.String" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from shenyu_dict where `type` = #{type,jdbcType=VARCHAR}
+        SELECT
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml:63]: The keywords `SELECT` are not right-aligned, it should be set back one space.

File: shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml [hunk 3]
```diff
@@ -32,65 +32,89 @@
...
     </select>
 
     <select id="findByTypeBatch" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from shenyu_dict where `type` in
-        <foreach collection="typeSet" item="type"
-                 open="(" separator="," close=")">
-            #{type,jdbcType=VARCHAR}
+        SELECT
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml:70]: The keywords `SELECT` are not right-aligned, it should be set back one space.

File: shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml [hunk 4]
```diff
@@ -32,65 +32,89 @@
...
+        <foreach collection="typeSet" item="type" open="(" separator="," close=")">
+              #{type,jdbcType=VARCHAR}
         </foreach>
     </select>
 
     <delete id="delete" parameterType="java.lang.String">
-        delete from shenyu_dict
-        where id = #{id,jdbcType=VARCHAR}
+        DELETE FROM shenyu_dict
+        WHERE id = #{id,jdbcType=VARCHAR}
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml:81]: The keywords `WHERE` are not right-aligned, It should be right-aligned with `FROM`.

**Comment by shallwetalkge** [comment]
@midnight2104 hi,I commited my change,can you review again,thanks

**Comment by midnight2104** [comment]
hi,pls check ci

**Comment by midnight2104** [comment]
Before committing, the master branch needs to be merged to ensure successful local execute `mvn clean install`.

File: shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml [hunk 5]
```diff
@@ -187,66 +211,71 @@
...
     </select>
 
     <update id="enabled">
-        update shenyu_dict set enabled = #{enabled,jdbcType=TINYINT} where id in
+        UPDATE shenyu_dict
+           SET enabled = #{enabled,jdbcType=TINYINT}
+         WHERE id IN
         <foreach item="item" index="index" collection="ids" open="(" separator="," close=")">
             #{item}
         </foreach>
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml:281]: good, pls check  ci

**Comment by shallwetalkge** [comment]
@midnight2104 Thank you for your patient guidance for my first pr

File: shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml [hunk 6]
```diff
@@ -32,65 +32,89 @@
...
+                    dict_value,
+                    `desc`,
+                    sort,
+                    enabled,
+                    date_created,
+                    date_updated)
+             VALUES (#{id,jdbcType=VARCHAR},
+                    #{type,jdbcType=VARCHAR},
+                    #{dictCode,jdbcType=VARCHAR},
+                    #{dictName,jbcType=VARCHAR},
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml:107]: pls modify `jbcType` ->  `jdbcType`

Comment (yu199195) [reply shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml:107]: pls modify jbcType -> jdbcType

Comment (shallwetalkge) [reply shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml]: sry,my mistake,I update it now

**Comment by midnight2104** [comment]
good job. If you want, you can add me on WeChat `LL332595980` and invite you to join the shenyu contributor group.

**Review by midnight2104 — APPROVED** [review]

**Pull request merged**